internal/pkg/fixtures/local/empty-branch/main/description.md

```diff
@@ -1 +0,0 @@
-Main branch
```

Comment:
Už nevím proč to tu bylo, ale potřeba to zřejmě není. Zbytečně se to přidávalo do vytvořenýho testu.

Reply:
ok, asi sa to skopirovalo z ineho testu